Vagrantfile

```diff
@@ -1,73 +1,108 @@
 # vi: set ft=ruby :
 
 require 'etc'
+require 'socket'
+
+if ENV['JENKINS'] == 'true'
+  ci_cluster = true
+  fqdn = Socket.gethostname
+  dot = fqdn.index('.')
+  hostname = fqdn[0..dot - 1]
+  domain = fqdn[dot + 1..-1]
+  mach_num = hostname[hostname.index("-") + 1..-1]
+  cluster_num = 0
+  if cluster_num == 0
+    cluster_num_s = ""
+  else
+    cluster_num_s = cluster_num
+  end
+else
+    ci_cluster = false
+    hostname = ""
+    cluster_num_s = ""
```

Comment:
if cluster_num is always 0 in the first if, you could just move cluster_num_s outside of the if statement.